parquet/src/test/java/org/apache/iceberg/parquet/TestVariantReaders.java

```diff
@@ -1327,7 +1327,7 @@ public void testArrayWithElementNullValueAndNullTypedValue() throws IOException
     VariantValue actualValue = actualVariant.value();
     assertThat(actualValue.type()).isEqualTo(PhysicalType.ARRAY);
     assertThat(actualValue.asArray().numElements()).isEqualTo(1);
-    assertThat(actualValue.asArray().get(0)).isNull();
+    VariantTestUtil.assertEqual(Variants.ofNull(), actualValue.asArray().get(0));
```

Comment:
I think this could be a simpler assertion by checking an expected array against the actual, but this is okay, too.